model/lesson.js

```diff
@@ -28,6 +29,22 @@ const attributes = {
  *          description: a URL to the lesson recording
  *          example: "https://www.facebook.com/watch/live/?ref=watch_permalink&v=244235014324418"
  */
+
+/**
+ *  @schema LessonInformation
```

Comment:
Would it make more sense to call this "Lesson content". Lesson info makes it seem like it's metadata, not actual content.

Reply:
If changed, please change everywhere (ctrl + shift + f then ctrl + shift + h  <--- find then replace)